webcams/transalpina.php

```diff
@@ -0,0 +1,13 @@
+<div class="webcam">
+    <span>Partia Constantinescu</span>
+    <div class="box">
+        <iframe src="http://82.79.247.31:888/cgi-bin/faststream.jpg?stream=full&fps=12" frameborder="0"></iframe>
+    </div>
+</div>
+
+<div class="webcam">
+    <span>Sosire Telescaun</span>
+    <div class="box">
```

Comment:
```suggestion
    <div>
```
No need for class as we're holding an image.